edb/protocol/enums.py

```diff
@@ -41,3 +43,18 @@ class Cardinality(enum.Enum):
 
     # Cardinality is >= 1
     AT_LEAST_ONE = 0x4d
+
+    @classmethod
+    def from_ir_value(cls, card: ir.Cardinality) -> Cardinality:
+        if card == ir.Cardinality.AT_MOST_ONE:
```

Comment:
Use `is` to test for enum (https://docs.python.org/3/library/enum.html#comparisons)

```suggestion
        if card is ir.Cardinality.AT_MOST_ONE:
```